test/KnoBlockIO.ts

```diff
@@ -73,87 +73,173 @@ describe('KnoBlockIO contract', function () {
     });
     describe('KnoBlock Deposit', function () {
```

Comment:
```suggestion
    describe('#deposit()', ()  => {
```

Follow pattern everywhere else as well.